framework/raw-aes-keyring.md

```diff
@@ -117,10 +117,10 @@ the [ciphertext](structures.md#ciphertext) field in
 The following table describes the fields that form the ciphertext for this keyring.
 The bytes are appended in the order shown.
 
-| Field              | Length (bytes)                                          | Interpreted as |
-| ------------------ | ------------------------------------------------------- | -------------- |
-| Encrypted Key      | length of AES-GCM ciphertext output                     | Bytes          |
-| Authentication Tag | [Authentication Tag Length](#authentication-tag-length) | Bytes          |
+| Field              | Length (bytes)                                                   | Interpreted as |
+| ------------------ | ---------------------------------------------------------------- | -------------- |
+| Encrypted Key      | length of AES-GCM ciphertext output                              | Bytes          |
+| Authentication Tag | [Authentication Tag Length](#authentication-tag-length) as Bytes | Bytes          |
```

Comment:
Authentication Tag Length represents bits, not bytes. This needs to be converted for the table to make sense.